library/test/dotty/ShowTests.scala

```diff
@@ -64,7 +64,7 @@ class ShowTests {
     case class Car(model: String, manufacturer: String, year: Int)
 
     assertEquals("Car(Mustang,Ford,1967)", Car("Mustang", "Ford", 1967).show)
-    assertEquals("Map()", Map().show)
+    assertEquals("Map()", Map[String, String]().show)
```

Comment:
What's the warning emitted?

The test actually expect something of type `Map[Nothing, Nothing]`.